lib/facilities/ppms/v1/configuration.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
```

Comment:
Looks like coverage is only at 68.18%

Reply:
because it's not being used without the missing change